web-wallet/src/routes/(app)/dashboard/staking/__tests__/__snapshots__/page.spec.js.snap

```diff
@@ -145,7 +145,6 @@ exports[`Staking > should render the staking page 1`] = `
         >
           <button
             class="dusk-button dusk-button--type--button dusk-button--variant--primary dusk-button--size--default dusk-icon-button--labeled contract-operations__operation-button"
-            disabled=""
```

Comment:
Seems like this is the stake button. Looking at the snapshot, the spendable is 0.000000000, so the stake should still be disabled? 👀